src/App.css

```diff
@@ -1,16 +1,10 @@
-.App {
-  text-align: center;
-}
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222045497-87c8ae3e-deda-4512-9d69-764771856fcb.png)

- [ ] Your branch name is not descriptive enough :fearful: because your branch name should tell what you are doing in this particular branch so it would be nice if you use **feature/add-calculator-components** instead of **calculator** to follow the professional practice and requirements of project :+1:

Reply:
- [ ] please check the project requirement this is an optional suggestion.

Reply:
Can you specify where this requirement is mentioned as an optional requirement :thinking: because I don't see anywhere that it is an optional requirement :arrow_down:

![image](https://user-images.githubusercontent.com/93039359/222051723-badcb5c8-7b5e-4f21-ab11-86398addf07a.png)

Reply:
- [ ] Ok Thank you.